### PR TITLE
Add multiple option to ceiling

### PR DESCRIPTION
**Pull request by optimumtact**
Thanks to @MrStonedOne for pointing this out

I guess og credit goes to PJB from vgstation, since they seem to have this implementation here
https://github.com/d3athrow/vgstation13/commit/1d92fdc37616ec75ce1cb9b5a66bd08325940ff4

### CODE REVIEW

**Comment by optimumtact** [comment]
hmm hold up, this could potentially round down, I'll need to test it

**Comment by Cheridan** [comment]
but the station doesn't have a ceiling

**Comment by optimumtact** [comment]
dontbelievehislies

File: code/__HELPERS/maths.dm [hunk 1]
```diff
@@ -17,8 +17,8 @@ var/list/sqrtTable = list(1, 1, 1, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 3, 4, 4, 4,
 	var/a = arccos(x / sqrt(x*x + y*y))
 	return y >= 0 ? a : -a
 
-/proc/Ceiling(x)
-	return -round(-x)
+/proc/Ceiling(x, y)
+	return -round(-x / y) * y
```
Comment (KazeEspada) [inline code/__HELPERS/maths.dm]: y defaults to 0, and it divides by y. This is a division by 0 runtime waiting to happen.

Comment (optimumtact) [inline code/__HELPERS/maths.dm]: Whoops, that's meant to be y=1

good catch

**Pull request merged**